src/components/atoms/Typography/style.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+import * as T from '../../../types/typography/typography';
+import { setSize, setColor, setWeight } from '../../../containers/helpers/typography/typography';
+
+export const Typography = styled.span<T.TypographyProps>`
```

Comment:
better return different element h1/h2/p/span depends on prop size

Reply:
Can you give me more details about this please ?
Did you mean add more style's for each element and when the typography with size X it will show the associated element ?